rapidtide/correlate.py

```diff
@@ -752,6 +776,7 @@ def arbcorr(
     method="univariate",
     debug=False,
 ):
+    """Calculate something."""
```

Comment:
I don't know what this function does.

Reply:
Calculates a correlation between under sampled signals.  Same function as implemented in the class above.  I'll accept the PR and then clean this up.